### PR TITLE
Support the UUID Column Type

### PR DESCRIPTION
**Pull request by arkie**

### CODE REVIEW

File: dataparser.go [hunk 1]
```diff
@@ -399,7 +399,7 @@ func newDataParser(t *TypeDesc, unquote bool) (DataParser, error) {
 		return &floatParser{32}, nil
 	case "Float64":
 		return &floatParser{64}, nil
-	case "String", "Enum8", "Enum16":
+	case "String", "Enum8", "Enum16", "UUID":
```
Comment (bgaifullin) [inline dataparser.go]: Please add tests.

Comment (arkie) [reply dataparser.go]: 👍

**Comment by bgaifullin** [comment]
LGTM, Please resolve conflicts with master and I will merge this PR

**Comment by arkie** [comment]
> Please resolve conflicts

@bgaifullin Branch should be up to date and mergeable - not sure what you mean?

**Comment by bgaifullin** [comment]
Github shows this message "This branch cannot be rebased due to conflicts".

**Comment by arkie** [comment]
Interesting - sounds like may be wonkiness around Github caching or similar, but saw you merged another PR so remerged master and it seems to have passed tests again.

I see:
image

**Comment by bgaifullin** [comment]
Seems like it was some glitch with github. 
Thank you for PR.

**Pull request merged**